From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: deb-examples

# DP: document readline header location on Debian systems

# DP: document readline header location on Debian systems
---
 examples/loadables/README | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/examples/loadables/README b/examples/loadables/README
index c729a6a..6902d29 100644
--- a/examples/loadables/README
+++ b/examples/loadables/README
@@ -40,6 +40,9 @@ rest of the example builtins. It's intended to be a start at something
 that can be modified or included to help you build your own loadables
 without having to search for the right CFLAGS and LDFLAGS.
 
+On Debian GNU/Linux systems, the bash headers are in /usr/include/bash.
+The appropriate options are already set in the example Makefile.
+
 basename.c	Return non-directory portion of pathname.
 cat.c		cat(1) replacement with no options - the way cat was intended.
 csv.c		Process a line of csv data and store it in an indexed array.
